src/sentry/options/defaults.py

```diff
@@ -1641,12 +1641,12 @@
 )
 register(
     "performance.issues.consecutive_http.problem-creation",
-    default=0.0,
+    default=1.0,
```

Comment:
[I added this option](https://github.com/getsentry/sentry/pull/90486/files#diff-4bcc09e0376432f8c06c7232afd12c430472607c1c2b0b0e53f7bfd43705054aL39-L40) to replace the previous ones: `performance.issues.[detector].flag_disabled`. These were not set in `options-automator`, so [the check here](https://github.com/getsentry/sentry/blob/master/src/sentry/utils/performance_issues/base.py#L100-L122) would always exit instead of creating any issues even though the detector was already running!